### DISCUSSION TITLE
PoE and Ethernet on one MicroUSB for V3 cam?

### DISCUSSION THREAD

**Discussion by KevinKadow** [gtxaspec/wz_mini_hacks#292] [General]
Anybody tried the [new Uctronics adapter](https://smile.amazon.com/dp/B0B9ZLVGJ4) on a V3 cameras as a way to provide Power-over-Ethernet (PoE) and data with a minimum of cables and adapters?   They claim this is improved over the original.

Released in mid-August, no reviews on Amazon.  I ordered one to test.

**Comment by RyanHope** [comment]
I ordered one at well to try out.

**Comment by KevinKadow** [comment]
Success.

 `[    3.072716] usbcore: registered new interface driver r8152
[    3.139473] r8152 1-1:1.0 eth0: v1.0.0 (2013/05/03)
`

Streaming RTSP into VLC is still somewhat glitchy, other RTSP viewers (e.g. TinyCam) are more stable than they were over WiFI

**Comment by plloppii** [comment]
What is your setup here?
I'm trying to connect these camera with my pepwave router with POE. 
I installed wz_mini_hacks on the camera and did ENABLE_USB_ETH="true", but im still getting a very spotty connection. 
It sometimes connects with eth, and some times my router just refuses to give it a dhcp lease.

**Reply by Bob-AOL** [reply]
The subject is covered extensively in the LAN Adapters discussion at the top page.

**Reply by plloppii** [reply]
Yup! I found that discussion last night. 
(https://github.com/gtxaspec/wz_mini_hacks/discussions/73)

If people are getting very mixed results on this adapter, might be worthwhile to increase visibility of the unreliability of this adapter in the README, as I know many people are looking for a POE + data solution for these camera.

**Reply by Bob-AOL** [reply]
The problems with these adapters have been addressed and they are now quite reliable.

**Comment by hacstest04f** [comment]
This seems like a great solution.  I need the POE part a few feet away from the camera itself.  Maybe use a microUSB female to male extension cable like this?

https://www.amazon.com/Sumind-Extension-Extender-Wireless-Security/dp/B07L9Q7581

**Reply by Bob-AOL** [reply]
They appear to be power only cables and will likely not transfer data if you are also going to be using the UCTRONICS adapter for ethernet connectivity.

These work well if you only need POE.

https://www.amazon.com/Wasserstein-Power-Adapter-Compatible-Outdoor/dp/B09H6SRS6C/ref=asc_df_B09H6SRS6C/?tag=hyprod-20&linkCode=df0&hvadid=563598683736&hvpos=&hvnetw=g&hvrand=6071085575087387558&hvpone=&hvptwo=&hvqmt=&hvdev=c&hvdvcmdl=&hvlocint=&hvlocphy=9015284&hvtargid=pla-1602929001035&psc=1
